packages/components/src/languageSupport/squiggle.ts

```diff
@@ -4,86 +4,134 @@ import {
-        Number: t.integer,
-        String: t.string,
-        Comment: t.comment,
-        Void: t.escape,
+              Syntax: t.annotation,
+              Boolean: t.bool,
+              Number: t.integer,
+              String: t.string,
+              Comment: t.comment,
+              Void: t.escape,
 
-        FunctionName: t.function(t.variableName),
+              FunctionName: t.function(t.variableName),
 
-        LambdaSyntax: t.blockComment,
+              LambdaSyntax: t.blockComment,
 
-        VariableName: t.constant(t.variableName),
-        Field: t.variableName,
-        LambdaParameter: t.variableName,
-      }),
-      foldNodeProp.add({
-        LambdaExpr: (context) => ({
-          from: context.getChild("NonEmptyProgram")?.from || 0,
-          to: context.getChild("NonEmptyProgram")?.to || 0,
+              VariableName: t.constant(t.variableName),
+              Field: t.variableName,
+              LambdaParameter: t.variableName,
+            }),
+            foldNodeProp.add({
+              LambdaExpr: (context) => ({
+                from: context.getChild("NonEmptyProgram")?.from || 0,
+                to: context.getChild("NonEmptyProgram")?.to || 0,
+              }),
+              BlockExpr: foldInside,
+              RecordExpr: foldInside,
+              ArrayExpr: foldInside,
+            }),
+            indentNodeProp.add({
+              RecordExpr: (context) =>
+                context.baseIndent +
+                (context.textAfter === "}" ? 0 : context.unit),
+              BlockExpr: (context) =>
+                context.baseIndent +
+                (context.textAfter === "}" ? 0 : context.unit),
+              LambdaExpr: (context) =>
+                context.baseIndent +
+                (context.textAfter === "}" ? 0 : context.unit),
+              ArrayExpr: (context) =>
+                context.baseIndent +
+                (context.textAfter === "]" ? 0 : context.unit),
+            }),
+          ],
         }),
-        BlockExpr: foldInside,
-        RecordExpr: foldInside,
-        ArrayExpr: foldInside,
-      }),
-      indentNodeProp.add({
-        RecordExpr: (context) =>
-          context.baseIndent + (context.textAfter == "}" ? 0 : context.unit),
-        BlockExpr: (context) =>
-          context.baseIndent + (context.textAfter == "}" ? 0 : context.unit),
-        LambdaExpr: (context) =>
-          context.baseIndent + (context.textAfter == "}" ? 0 : context.unit),
-        ArrayExpr: (context) =>
-          context.baseIndent + (context.textAfter == "]" ? 0 : context.unit),
-      }),
-    ],
-  }),
-  languageData: {
-    commentTokens: {
-      line: "#",
-    },
-    autocomplete: [
-      snippetCompletion("{ |${args}| ${body} ", {
-        label: "{",
-        detail: "lambda function",
-      }),
-    ],
-    closeBrackets: {
-      brackets: ['"', "'", "(", "{"],
-    },
-  },
-});
+        languageData: {
+          commentTokens: {
+            line: "#",
```

Comment:
I'm unsure if this affects anything, but: `//` is also a line comment, and `#` will probably be deprecated eventually.

Reply:
It's used for Ctrl-/, so probably we're better off switching it to `//` if `#` is to go. 
Oof, deprecating comment symbols in a language.